torch/csrc/api/include/torch/nn/options/normalization.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <torch/arg.h>
+#include <torch/csrc/WindowsTorchApiMacro.h>
+#include <torch/types.h>
+
+namespace torch {
+namespace nn {
+
+/// Options for the `Normalize` module.
+struct TORCH_API NormalizeOptions {
+  /// The exponent value in the norm formulation. Default: 2
+  TORCH_ARG(double, p) = 2;
```

Comment:
nit: would be great to write:
```suggestion
  TORCH_ARG(double, p) = 2.0;
```
to better indicate that it's a double value.